src/api/AutonomousCars.Api/Program.cs

```diff
@@ -25,6 +25,7 @@ private static void Main(string[] args)
 
         // Options
         builder.Services.Configure<AzureMapsOptions>(builder.Configuration.GetSection("AzureMaps"));
+        builder.Services.Configure<MqttNamespaceOptions>(builder.Configuration.GetSection("MqttNamespace"));
```

Comment:
Ici tu dois pouvoir ajouter de la validation si tu veux vraiment vérifier qu'on a bien toute la configuration (pas obligé de t'embêter à faire ça). L'utilisation de Options convient très bien ici, par contre la configuration tu la mets via les user secrets ce qui veut dire que ça ne marchera qu'en local et pas sur Azure. Donc il faudrait plutôt ajouter les secrets dans un [Azure Key Vault de manière à ce qu'ils soient disponibles dans la configuration](https://learn.microsoft.com/en-us/aspnet/core/security/key-vault-configuration?view=aspnetcore-8.0#use-managed-identities-for-azure-resources).